### PR TITLE
fix(ColPinning): #4503 Set min-width to dropdown.

### LINKED ISSUE #4503: Pinning dropdown layout is broken in Grid Toolbar
## Description  

 * igniteui-angular version: 7.2.x
 * browser: all

## Steps to reproduce  

1. Open the following [sample](https://stackblitz.com/edit/angular-skrbx5)
2. Open pinning dropdown
3. Type randon string in order to not have results
4. Click somewhere to close the dropdown
5. Try to open the pinning dropdown

## Result  
image

## Expected result  

image

### PR DESCRIPTION
**Pull request by gedinakova**
Closes #4503   

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
                 <span></span>
             </div>
         </button>
-        <igx-drop-down #columnPinningDropdown>
+        <igx-drop-down #columnPinningDropdown [style.min-width.px]="180">
```
Comment (kdinev) [inline projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html]: I think this should be done with a class, or inside the toolbar theme. @simeonoff

Comment (simeonoff) [reply projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html]: @kdinev should this be a global setting for the drop-down menu? Like the drop-down should not go bellow 180 px?

Comment (kdinev) [reply projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html]: @simeonoff I'm not sure. Is there a guideline for this in material?

Comment (simeonoff) [reply projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html]: @kdinev Yes, there is a min-width of 128px for the drop down menus.

Comment (kdinev) [reply projects/igniteui-angular/src/lib/grids/grid-toolbar.component.html]: @simeonoff Makes sense. Implement it globally then!

**Pull request merged**